docs/manual/getting_started.rst

```diff
@@ -1516,12 +1852,14 @@ when managing data to lower overhead.
 System requirements
 ~~~~~~~~~~~~~~~~~~~
 
+Before You Get Started with Eclipse Cyclone DDS
+===============================================
+
 At the time of writing this document, Eclipse Cyclone DDS CXX supports
```

Comment:
I've already seen this content earlier in the document, it looks to be duplicated.